nodejs/awsx/elasticloadbalancingv2/loadBalancer.ts

```diff
@@ -138,3 +153,52 @@ export interface LoadBalancerSubnets {
 function isLoadBalancerSubnets(obj: any): obj is LoadBalancerSubnets {
     return obj && (<LoadBalancerSubnets>obj).subnets instanceof Function;
 }
+
+export interface LoadBalancerTarget {
+    /**
+     * The ID of the target. This is the Instance ID for an `instance`, or the container ID for an
+     * ECS container. If the target type is `ip`, specify an IP address. If the target type is
+     * `lambda`, specify the arn of lambda.
+     */
+    targetId: string;
+    /**
+     * The Availability Zone where the IP address of the target is to be registered.
+     */
+    availabilityZone?: string;
+    /**
+     * The port on which targets receive traffic.
+     */
+    port?: number;
+}
+
+export interface LoadBalancerTargetProvider {
+    loadBalancerTarget(targetType: pulumi.Input<mod.TargetType>): pulumi.Output<LoadBalancerTarget>;
+}
+
+export function isLoadBalancerTargetProvider(obj: any): obj is LoadBalancerTargetProvider {
+    return (<LoadBalancerTargetProvider>obj).loadBalancerTarget instanceof Function;
+}
+
+/**
+ * Allows an EC2 instance to simply be used as the target of an ALB or NLB.  To use, just call:
+ *
+ * ```ts
+ *  lb.attachTarget(new Ec2InstanceTarget(instance));
+ * ```
+ */
+export class Ec2InstanceTarget implements LoadBalancerTargetProvider {
```

Comment:
helper class to turn an aws.ec2.Instance into a LoadBalancerTargetProvider.  Won't be necessary if we can just monkeypatch aws.ec2.Instance to expose the `loadBalancerTarget` method.